sample-compose/data/src/main/java/co/nimblehq/sample/compose/data/response/Response.kt

```diff
@@ -4,9 +4,15 @@ import co.nimblehq.sample.compose.domain.model.Model
 import com.squareup.moshi.Json
 
 data class Response(
-    @Json(name = "id") val id: Int?
+    @Json(name = "id")
+    val id: Int?,
+    @Json(name = "username")
+    val userName: String?,
```

Comment:
`username` is one word.
```suggestion
    val username: String?,
```

Reply:
@AVI5HEK Maybe it's an exception here to produce the sample test with a differ between field name and JSON key name 😛

Reply:
@luongvo Can we have a comment to explain the sample here too? I'm concerned that after this PR, someone will update it to `username` as we don't know the purpose of a differ name example.

Reply:
@lydiasama Yes, we should indeed.

Reply:
@AVI5HEK @lydiasama Added in https://github.com/nimblehq/android-templates/pull/480/commits/991c23356f8d3add114f86ff6832d905e51efa07